system_setup/common/wsl_conf.py

```diff
@@ -74,9 +80,20 @@ def wsl_config_loader(data, config_ref, id):
             conf_sec_list = config[conf_sec]
             for conf_item in conf_sec_list:
                 if conf_item in config_ref[id][conf_sec]:
+                    # Handle systemd experimental feature (string -> bool)
+                    if (
+                            conf_sec.lower() == "boot" and
+                            conf_item.lower() == "command"):
```

Comment:
Nitpick: I'm not quite sure why this line break after the `if(` clause.

Reply:
That was for the linter to not complain, let’s see what other says, I dno’t really like it either (but 80 characters, avoid alignements errors…)